sunbeam-python/sunbeam/plugins/validation/plugin.py

```diff
@@ -456,20 +470,16 @@ def configure_validation(self, options: Optional[List[str]] = None) -> None:
     )
     def run_validate_action(
         self,
-        regex: str = "",
-        exclude_regex: str = "",
-        serial: bool = False,
-        test_list: str = "",
-        output: Optional[str] = None,
+        profile: str,
+        output: Optional[str],
     ) -> None:
-        """Run a set of tests on the sunbeam installation."""
+        """Run tests against the cloud ("refstack" profile by default).
+
+        PROFILE is the set of tests to run.
+        For details of available profiles, run `sunbeam validation profiles`.
+        """
```

Comment:
Can this be changed to something like:
```
"""Run tests against the cloud ("refstack" profile by default).

Arguments:
    <profile> The set of tests to run. For details of available profiles, run `sunbeam validation profiles`.
"""
```
This way we can have a section for this positional argument, similar to Options.

Reply:
Thanks for updating this. The current result already looks better, but I was hoping the argument section can be formatted the same way as (or as close as possible to) `Options`:

```
Usage: sunbeam validation run [OPTIONS] [PROFILE]

  Run validation tests (default: "refstack" profile).

Arguments:
  [PROFILE]        The set of tests to run (defaults to "refstack"). For details of available profiles, run `sunbeam validation profiles`.

Options:
  -o, --output PATH     ...
```

Some what a nit, but I think it will improve readability.

Reply:
There is a way to format custom help message, but probably a bit overkill. My comment was not meant to be a blocker, and I think the current approach is good enough at this stage.